### PR TITLE
Cleanup cc stubs: ccMenuCallback => menuCallback

### PR DESCRIPTION
**Pull request by aismann**
## Describe your changes

Rename ax::ccMenuCallback to ax::menuCallback

### For each PR
- [x] Add Copyright if it missed:   
      - `"Copyright (c) 2019-present Axmol Engine contributors (see AUTHORS.md)."`
- [x] I have performed a self-review of my code.
       
   Optional:
   - [x] I have checked readme and add important infos to this PR.
   - [x] I have added/adapted some tests too.
          
### For core/new feature PR
- [ ] I have checked readme and add important infos to this PR.
- [ ] I have added thorough tests.

### CODE REVIEW

File: core/2d/MenuItem.cpp [hunk 1]
```diff
@@ -148,7 +149,7 @@ void MenuItemLabel::setLabel(Node* var)
     _label = var;
 }
 
-MenuItemLabel* MenuItemLabel::create(Node* label, const ccMenuCallback& callback)
+MenuItemLabel* MenuItemLabel::create(Node* label, const menuCallback& callback)
```
Comment (halx99) [inline core/2d/MenuItem.cpp:152]: the type shoud starts with uppercase

Comment (TyelorD) [reply core/2d/MenuItem.cpp:152]: Even though menuCallback is just a typedef alias for `std::function<void(Ref*)>`?

Comment (halx99) [reply core/2d/MenuItem.cpp:152]: maybe use common callback is better

Comment (halx99) [reply core/2d/MenuItem.cpp:152]: maybe: InputCallback

Comment (aismann) [reply core/2d/MenuItem.cpp:152]: There be 3 same typedef
image
What is your proposal?

Comment (smilediver) [reply core/2d/MenuItem.cpp:152]: I vote for just getting rid of `cc` prefix, ie leave `MenuCallback`, `WidgetClickCallback`, etc.

**Comment by aismann** [comment]
I closed it. need a deeper look

**Pull request closed without merging**